react-common/components/profile/Profile.tsx

```diff
@@ -44,5 +46,24 @@ export const Profile = (props: ProfileProps) => {
             userState={userBadges}
             onBadgeClick={onBadgeClick}
         />}
+
+        {profileSmall &&
+            <div className="profile-info-container">
+                <p className="profile-info">
+                    {lf("Now that you're logged in, your projects will be automatically saved to the cloud so you can access them from any device! ")}
+                    {lf("Learn more at ")}
```

Comment:
We can't concat strings like this to form a sentence, it becomes untranslatable due to languages having different sentence structures. there is a helper here https://github.com/microsoft/pxt/blob/glueInDepsForHex/react-common/components/util.tsx#L15 that can be used like this https://github.com/microsoft/pxt/blob/glueInDepsForHex/skillmap/src/components/AppModal.tsx#L581